src/SFA.DAS.AssessorService.Api.Types/Models/CertificateAddressResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using MediatR;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models
+{
+    public class CertificateAddressResponse : IRequest
```

Comment:
In fact, do we need this class at all as it's nigh on identical to CertificateAddress? Maybe just return CertificateAddress all the way through and then there's no need for the Mapping.

Reply:
Yep wasn't sure about passing the view model out of the repository. If your happy with this I have no issues -

Reply:
Yeah, there's nothing wrong with the Entity making it's way all the way out of the API. 👍